src/azul/service/responseobjects/step_function_helper.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from azul.deployment import aws
+
+
+class StepFunctionHelper:
+    """
+    Wrapper around boto3 SFN client to handle resource name generation and state machine executions
+    """
+    def state_machine_arn(self, state_machine_name):
+        return f'arn:aws:states:{aws.region_name}:{aws.account}:stateMachine:{state_machine_name}'
+
+    def execution_arn(self, state_machine_name, execution_name):
+        return f'arn:aws:states:{aws.region_name}:{aws.account}:execution:{state_machine_name}:{execution_name}'
+
+    def start_execution(self, state_machine_name, execution_name=None, execution_input=None):
+        """
+        Start an execution of a state machine
+        Wrapper around https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/stepfunctions.html#SFN.Client.start_execution
+
+        :param state_machine_name: name of state machine to run
+        :param execution_name: name to give execution; random uuid is used if name is not given
+        :param execution_input: Input to execution that will be JSON-encoded
+        :return: Dict with structure
+            {
+                'executionArn': 'string',
+                'startDate': datetime(2015, 1, 1)
+            }
+        """
+        execution_params = {
+            'stateMachineArn': self.state_machine_arn(state_machine_name),
+            'input': json.dumps(execution_input)
+        }
+        if execution_name is not None:
+            execution_params['name'] = execution_name
+        return aws.stepfunctions.start_execution(**execution_params)
```

Comment:
The return value isn't actually used anywhere so this function should return nothing. Less to test, less to document, less to worry about.

Reply:
The return value is used to get the execution id which is used to create the retry token

Reply:
We have the ARN of the state machine but not of the execution of the state machine that we just created.  Name is an ID but the official documentation calls it execution name

Reply:
I prefer this over returning a value:

```suggestion
        assert self.execution_arn(state_machine_name, execution_name) == aws.stepfunctions.start_execution(**execution_params)['executionArn']
```

Just not all on one line.